Model/Api/CreateOrder.php

```diff
@@ -536,7 +536,7 @@ public function hasItemErrors($quoteItem)
         $errorInfos = $quoteItem->getErrorInfos();
         $message = '';
         foreach ($errorInfos as $errorInfo) {
-            $message .= '(' . $errorInfo['origin'] . '): ' . $errorInfo['message']->render() . PHP_EOL;
+            $message .= '(' . $errorInfo['origin'] . '): ' . (is_string($errorInfo['message']) ?  $errorInfo['message'] : $errorInfo['message']->render()) . PHP_EOL;
```

Comment:
Out of curiosity, what can `$errorInfo['message']` be that's not a string?

Reply:
I'm not 100% sure as I haven't seen anything except the string type here, but I think it can be an object that can that has a method named render. @YevhenShBolt Since you wrote that code in the first place, can you confirm?

Reply:
I'm not 100% sure as I haven't seen anything except the string type here, but I think it can be an object that has a method named render. @YevhenShBolt Since you wrote that code, can you confirm?